platform/ios/jazzy.yml

```diff
@@ -23,55 +26,62 @@ custom_categories:
       - MGLMapViewDelegate
       - MGLStyle
       - MGLUserTrackingMode
-  - name: Annotations
+  - name: Protocols and Abstract Classes
```

Comment:
I wonder if this category’s title still makes sense given that some abstract classes are in more specific categories below.